src/coord_finder.cpp

```diff
@@ -93,35 +108,34 @@ class CoordFinder {
         double steps = (laser_angle_max * 2) / laser_angle_increment; 
         int index1 = (int)(((index1_angle + (laser_angle_max - (M_PI/2))) / (laser_angle_max*2))* steps);
         int index2 = (int)(((index2_angle + (laser_angle_max - (M_PI/2))) / (laser_angle_max*2))* steps);
-        ROS_DEBUG_STREAM("Coord Finder: index1 :" << index1 << " index2: " << index2);
-        ROS_DEBUG_STREAM("Coord Finder: size of scan message ranges " << scan_msg.ranges.size());
+        ROS_DEBUG_STREAM(TAG << "index1 :" << index1 << " index2: " << index2);
+        ROS_DEBUG_STREAM(TAG << "size of scan message ranges " << scan_msg.ranges.size());
+
         // check that the range indexes are within the range of the scan message and that index1 > index2
         if (index1 < 0 || index2 < 0 || index1 >= scan_msg.ranges.size() || index2 >= scan_msg.ranges.size() || index1 >= index2) {
-            ROS_WARN("PropInProgress message range indexes are out of bounds for the given scan message");
+            ROS_WARN_STREAM(TAG << "PropInProgress message range indexes are out of bounds for the given scan message");
             return;
         }
 
-        //create a 2D vector containing distance angle pairs for points detected by lidar within the range provided by yolo
-               //starting angle for lidar scan 
-        ROS_DEBUG_STREAM("Laser angle min" << laser_angle_min);
-        ROS_DEBUG_STREAM("Laser angle increment" << laser_angle_increment);
-        double starting_angle = laser_angle_min + (M_PI/2.0);
+        //create a 2D vector containing distance angle pairs for points detected by lidar      
+        ROS_DEBUG_STREAM(TAG << "Laser angle min" << laser_angle_min);
+        ROS_DEBUG_STREAM(TAG << "Laser angle increment" << laser_angle_increment);
+        double starting_angle = laser_angle_min + (M_PI/2.0); //starting angle for lidar scan 
         std::vector<lidarPoint> scanPoints = CoordFinder::createLidarPoints(scan_msg.ranges, starting_angle, laser_angle_increment);
         if (scanPoints.size()<1){
-            ROS_WARN("No points added to scanPoints vector");
+            ROS_WARN_STREAM("No points added to scanPoints vector");
```

Comment:
I notied some logs are using ```Tag``` and some arn't. Should they all use ```Tag```? Also what is Tag?

Reply:
TAG is just a label I added so we know which node printed that message. I did miss that one though.